documentation/modules/exploit/windows/smtp/sysgauge_client_bof.md

```diff
@@ -0,0 +1,39 @@
+## Vulnerable Application
+
+  This module will setup a SMTP server expecting a connection from SysGauge 1.5.18
+via its SMTP server validation. The module sends a malicious response along in the
+220 service ready response and exploits the client resulting in an unprivileged shell.
+
```

Comment:
I'd add something like: The software is available for download from [SysGauge](http://www.sysgauge.com/setups/sysgauge_setup_v1.5.18.exe).